src/_P150_TMP117.ino

```diff
@@ -139,24 +169,47 @@ boolean Plugin_150(uint8_t function, struct EventStruct *event, String& string)
           P150_CYCLE_8_SEC,
           P150_CYCLE_16_SEC,
         };
-        addFormSelector(F("Conversion cycle time"), F("cycle"), 8, cycleCaptions, cycleOptions, P150_GET_FLAG_CYCLE_BITS);
+        addFormSelector(F("Conversion cycle time"), F("cycle"), 8, cycleCaptions, cycleOptions, P150_GET_CONF_CYCLE_BITS);
         addFormNote(F("Not all Cycle time options are available for One-shot mode!"));
       }
 
+      addFormSubHeader(F("Output"));
+
+      addFormCheckBox(F("Temperature in Fahrenheit"), F("fahr"), P150_GET_OPT_FAHRENHEIT);
```

Comment:
Not really sure if this should be part of this plugin.
It can be useful for any temperature measurement and I don't think we should have this on some plugins and not all.

So this can also be done easily in the formula field.

Reply:
I'll remove that option.